packages/shared/trade/src/transaction-approvals/hooks/use-transaction-approvals.tsx

```diff
@@ -0,0 +1,64 @@
+import { useTokenApproval } from './use-token-approval';
+import { useEnablePrimeBorrow } from './use-enable-prime-borrow';
+import {
+  useAccountReady,
+  BaseTradeContext,
+  useWalletBalanceInputCheck,
+} from '@notional-finance/notionable-hooks';
+
+export const useTransactionApprovals = (
+  symbol: string,
+  context: BaseTradeContext,
+  isWithdraw?: boolean
+) => {
+  const {
+    state: { depositBalance, debt },
+  } = context;
+  const isAccountReady = useAccountReady();
+  const {
+    tokenStatus,
+    isSignerConnected,
+    enableToken,
+    tokenApprovalTxnStatus,
+  } = useTokenApproval(symbol);
```

Comment:
`symbol` is always passed in as `selectedDepositToken || ''` so might as well just do that here and remove the parameter.